herbert_core/classes/MPIFramework/@parallel_config/parallel_config.m

```diff
@@ -215,22 +215,19 @@
         %-----------------------------------------------------------------
         % overloaded getters
         function wrkr = get.worker(obj)
-            wrkr= get_or_restore_field(obj,'worker');
+            wrkr = get_or_restore_field(obj, 'worker');
```

Comment:
Is this a method and thus does this want to be called like a method?
`obj.get_or_restore_field(...)`